### ISSUE TITLE
[Bug]: Site closes when opening

### ISSUE THREAD

**Issue by zyrouge** [actualbudget/actual#2596]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue
- [X] I will be providing steps how to reproduce the bug (in most cases this will also mean uploading a demo budget file)

### What happened?

The web app closes when attempting opening. It occurs very frequently to me. It is just a plain instance with zero data. I've attached a clip of it happening. This has happened for me on Firefox (desktop) and on Brave (mobile).

Something similar to `history.back()` seems to be happening. I'm willing to open a PR if you could guide where it is happening.

https://github.com/actualbudget/actual/assets/50770114/2ab8b271-c4bd-493f-a4ad-76d5d82fd848

### Where are you hosting Actual?

Docker

### What browsers are you seeing the problem on?

Firefox

### Operating System

Linux

**Comment by shall0pass** [comment]
Are you using an authentication front end that might be redirecting you?  If you open the developers console, are there any errors or indication of what is happening?  I've never experienced anything like this, and over the past couple of years there haven't been any reports like this.  I have a feeling this is specific to your setup.

**Comment by zyrouge** [comment]
> Are you using an authentication front end that might be redirecting you? If you open the developers console, are there any errors or indication of what is happening? I've never experienced anything like this, and over the past couple of years there haven't been any reports like this. I have a feeling this is specific to your setup.

No authentication, no errors in console and also no redirects in networks. It does seem very weird to me.

**Comment by MatissJanis** [comment]
👋 How can we reproduce this issue?

I've never seen this issue before on my instance, so I wonder if this is actually something to do with the way how you host the app.

**Comment by zyrouge** [comment]
I used podman to run the image. After setting the password, I just closed the site. Re-opening the plain instance might close the site. This occured both on my local PC and my VPS.

**Comment by zyrouge** [comment]
Did some testing and could confirm that it only occurs in an empty instance. This issue disappeared when I added a account.

**Comment by MatissJanis** [comment]
My guess would be that [this line](https://github.com/actualbudget/actual/blob/16df116d1dceeb2c9603564249895341292a99c9/packages/desktop-client/src/hooks/useNavigate.ts#L40-L41) is causing the issue. But I'm not sure why.. and haven't been able to reproduce.

**Comment by MatissJanis** [comment]
Managed to reproduce on https://demo.actualbudget.org

1. open https://demo.actualbudget.org and create an empty budget
2. close the tab
3. open https://demo.actualbudget.org/accounts
4. close the tab
5. open https://demo.actualbudget.org/accounts
6. 💣

**Comment by MatissJanis** [comment]
cc @joel-jeremy 

Do you think it would cause problems if we removed [this line](https://github.com/actualbudget/actual/blob/16df116d1dceeb2c9603564249895341292a99c9/packages/desktop-client/src/hooks/useNavigate.ts#L40-L41)?

**Comment by matt-fidd** [comment]
@MatissJanis I can't reproduce this anymore, are you able to?

**Comment by MatissJanis** [comment]
No; seems to be resolved on my end. @zyrouge can you verify if it is resolved for you too?

**Comment by zyrouge** [comment]
@MatissJanis Seems to be fixed!
